apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java

```diff
@@ -192,9 +193,11 @@ public URL nextElement() {
                 for (String fileName : jarFileNames) {
                     try {
                         File file = new File(path, fileName);
-                        Jar jar = new Jar(new JarFile(file), file);
-                        jars.add(jar);
-                        LOGGER.info("{} loaded.", file.toString());
+                        try(var jarFile = new JarFile(file)){
```

Comment:
JarFile need to be resident in memory. @wu-sheng @acurtain

Reply:
`var` is not acceptable in the JDK8 compiler. Why use this?

Reply:
There will be no effect, because the following is "new Jar(jarFile, file);"
不会有影响, 因为后面是"new Jar(jarFile, file);"

Reply:
We don't recommend abusing `lombok`. Its annotation-based usages are enough. Don't put `var` in the codes.

Reply:
We are using Java, rather than Scala or Kotlin today, we don't want to make codes simpler, but want to make the codes easier to read for everyone. At here, you are just trying to fix the muse recommendation, so, please don't bring any related thing in.

Reply:
is false positive, I have restored the original code